### PR TITLE
[feature-2-impr] Create additional-file.txt

### PR DESCRIPTION
**Pull request by epleisman**
https://github.com/epleisman-grid/pull-requests/issues/5

Feature-2 was improved

### CODE REVIEW

File: feature-2/additional-file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This file improves the functionality of the feature 2
```
Comment (epleisman) [inline feature-2/additional-file.txt:1]: Great job!

Comment (epleisman-grid) [inline feature-2/additional-file.txt:1]: Great job

Comment (epleisman) [inline feature-2/additional-file.txt:1]: Great job!

Comment (epleisman-grid) [inline feature-2/additional-file.txt:1]: Great job man!

**Review by epleisman-grid — APPROVED** [review]
Great job!

**Pull request merged**